draft-ietf-rtcweb-jsep.xml

```diff
@@ -3481,13 +3481,13 @@ as specified in <xref target="RFC5245" format="default"/>.</li>
 	      section using that SSRC, as described in
 	      <xref target="RFC9143" sectionFormat="comma" section="9.2"/>.</li>
 		<li>For each specified RTP header extension that is also
-	      supported by the local implementation, establish a mapping
+	      enabled in the local implementation, establish a mapping
 	      between the extension ID and URI, as described in
 	      <xref target="RFC5285" sectionFormat="comma" section="5"/>. Specifically, this
 	      means that the implementation records the extension ID to
 	      be used in outgoing RTP packets when sending each specified
 	      header extension. If any indicated RTP header extension is
-	      not supported by the local implementation, it <bcp14>MUST</bcp14> be
+	      not enabled in the local implementation, it <bcp14>MUST</bcp14> be
```

Comment:
this one can remain "supported by" since it means "not implemented"

Reply:
yeah, I wondered about that, but shouldn't `enabled` also factor in here? That is, if somehow we get a SDP with a header extension that isn't enabled, shouldn't that header extension ID be ignored?